doc/api/assert.md

```diff
@@ -632,21 +632,26 @@ changes:
 > Stability: 0 - Deprecated: Use `assert.fail([message])` or other assert
 > functions instead.
 
-If `message` is falsy, the error message is set as the values of `actual` and
-`expected` separated by the provided `operator`. If just the two `actual` and
-`expected` arguments are provided, `operator` will default to `'!='`. If
-`message` is provided as third argument it will be used as the error message and
-the other arguments will be stored as properties on the thrown object. If
-`stackStartFn` is provided, all stack frames above that function will be
-removed from stacktrace (see [`Error.captureStackTrace`]). If no arguments are
-given, the default message `Failed` will be used.
+If just the two `actual` and `expected` arguments are provided, `operator` will
+default to `'!='`. If `message` is provided as third argument it will be used
+as the error message and the other arguments will be stored as properties on
+the thrown object. If `stackStartFn` is provided, all stack frames above that
+function will be removed from stacktrace (see [`Error.captureStackTrace`]).
+If no arguments are given, the default message `Failed` will be used.
+
+If `message` is falsy and just the three arguments provided, the
```

Comment:
I'm not sure it will be clear to readers what is meant by "just the three arguments"?

Reply:
```suggestion
If `message` is provided but falsy, and no other optional arguments are specified, the
```